docs/topic-guides/blendable-web-apps/view-model-attaching.md

```diff
@@ -64,7 +64,21 @@ Handlers with empty tokens are called with other handlers with the same empty to
 
 ## Contexts
 
-Token matching can be made more fine-grained by using contexts. They are composed of a list of strings that acts as a bit map when matched with other contexts. No context \(`null` value\) means **match any context**. Otherwise, two handlers are matched if source context contains same elements as destination context. Examples:
+Token matching can be made more fine-grained by using contexts. They are composed of a list of strings that acts as a bit map when matched with other contexts. This list isn't materialized anywhere, it is just an un-written contract between the apps.
+
+Starcounter came up with few predefined contexts that have shared meaning which can be understood by all app authors:
+* `page` — an entire page content;
+* `thumbnail` — a thumbnail of a page content;
+* `row` — single row of entity data;
+* `icon` — launch icons or smaller icons in other scenarios;
+* `search` — a search result;
+* `app` — a meta context that represents the current app;
+
+As an app author or solution owner, you can come up with his own contexts. That such contexts are unknown to other app developers, so they are less likely to attach view-models from other apps.
```

Comment:
As an app author or solution owner, you can come up with your own contexts. Keep in mind that such contexts are unknown to other app authors, so they are less likely to attach view-models from other apps.